src/ui/pages/SendForm/fieldsets/AddressInput/AddressInput.tsx

```diff
@@ -357,9 +353,15 @@ export function AddressInput({
             </UnstyledButton>
             <UIText kind="headline/h3">
               <UnstyledInput
-                autoFocus={autoFocus}
                 {...getInputProps({
                   ...inputProps,
+                  // use onChange instead of onInputValueChange for controlled inputs
+                  // https://github.com/downshift-js/downshift/issues/1108#issuecomment-674180157
+                  onChange: (e) => {
+                    setShowAllItems(false);
```

Comment:
should this line still be in `onInputValueChange`?

Reply:
Interesting question. We will probably need to combine props for onChange then